util/main_test.go

```diff
@@ -128,57 +128,74 @@ func TestGetFile(t *testing.T) {
 	}
 }
 
-func TestParseOgInfo(t *testing.T) {
+func TestParseAssignmentOgInfo(t *testing.T) {
```

Comment:
should also test precedence with flags. Maybe add a function that accepts an AssignmentInfo pointer and flag values, updates the pointer with flag values if necessary?

Reply:
helper function made